content/Coalesce.Starter.Vue.Web/Program.cs

```diff
@@ -91,14 +89,13 @@
 app.UseAuthentication();
 app.UseAuthorization();
 
-var containsFileHashRegex = new Regex(@"\.[0-9a-fA-F]{8}\.[^\.]*$", RegexOptions.Compiled);
 app.UseStaticFiles(new StaticFileOptions
 {
     OnPrepareResponse = ctx =>
     {
         // vite puts 8-hex-char hashes before the file extension.
         // Use this to determine if we can send a long-term cache duration.
-        if (containsFileHashRegex.IsMatch(ctx.File.Name))
+        if (ContainsFileHash().IsMatch(ctx.File.Name))
         {
             ctx.Context.Response.GetTypedHeaders().CacheControl =
                 new CacheControlHeaderValue { Public = true, MaxAge = TimeSpan.FromDays(30) };
```

Comment:
Since we're updating to new language features, can you change both instantiations of CacheControlHeaderValue to use target-typed `new`?

Reply:
Done